src/test/java/teammates/test/pageobjects/FeedbackSubmitPage.java

```diff
@@ -125,4 +125,8 @@ public void linkOnCommentsLink() {
         AppPage.getNewPageInstance(browser, StudentCommentsPage.class);
         
     }
+    
+    public void waitForCellHoverToDisappear() {
+        waitForElementToDisappear(By.className("cell-hover"));
```

Comment:
I don't think this works? The element with the "cell-hover" class never disappears, it is the "cell-hover" class that is removed from the element.